src/configurations/adknowledgeportal/synapseConfigs/studies.ts

```diff
@@ -81,6 +81,7 @@ export const studiesDetailsPageProps: DetailsPageProps = {
   tabLayout: [
     {
       title: 'Study Details',
+      uriValue: 'StudyDetails',
```

Comment:
These are DetailsPage tabs, so navigating to the tab will give a URL like /Explore/Studies/DetailsPage/StudyDetails?key=value

Reply:
The DetailsPageTabs component has a built-in redirect that should handle that particular case